src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -62,15 +62,42 @@ public void storeMessage(Message message) {
             .addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
+    // pMessage(messages, user, results);
```

Comment:
If you aren't using this, then I would remove it since other people are also writing code in this repository, they will be confused why there is a comment with code here.